modules/calcite/src/test/sql/subquery/any_all/test_uncorrelated_all_subquery.test

```diff
@@ -123,6 +132,14 @@ NULL	false
 2	false
 3	false
 
+query IT
+SELECT i, i <> ANY(SELECT i FROM integers WHERE i>10) FROM integers ORDER BY i NULLS FIRST;
```

Comment:
Does it actually work? The result seems to be `Actual:   [[1, false], [2, false], [3, false], [null, false]]` . No nulls-first